static/app/views/projectInstall/createProject.tsx

```diff
@@ -82,6 +86,18 @@ function CreateProject() {
     undefined
   );
 
+  const {
+    createNotificationAction,
+    actions: alertNotificationActions,
+    provider,
+    integration,
+    channel,
+    setActions: setAlertNotificationActions,
+    setProvider,
+    setIntegration,
+    setChannel,
```

Comment:
Do you actually need to destructure all these? Are you able to just have it look like this
```suggestion
    ...notificationProps,
```